<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pdf2pacs</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pdf2pacs.ico</normaloff>pdf2pacs.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Studienbeschreibung:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="studiendatum"/>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="studienbeschreibung"/>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Studiendatum:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nachname:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="dropArea" name="buttonLoadPDF">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 4px dashed #aaa</string>
      </property>
      <property name="text">
       <string>PDF Datei laden ...</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="nachname"/>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="buttonSearchPACS">
      <property name="text">
       <string>Im PACS suchen</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="aufnahmezeit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Vorname:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Serienbeschreibung:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="vorname"/>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QPushButton" name="buttonSendPACS">
      <property name="text">
       <string>Senden ins PACS</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLineEdit" name="serienbeschreibung"/>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Aufnahmezeit:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QTableWidget" name="table">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Nachname</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Vorname</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Patienten ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Studiendatum</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Studiebeschreibung</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>dropArea</class>
   <extends>QPushButton</extends>
   <header>pdf2pacs</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
